backend/utils/default_settings.json

```diff
@@ -2,8 +2,10 @@
   "version": "0.3.0",

```

Comment:
We need to be raising the version number on updates. Also, should have version number displayed in log during initialization